src/components/shared/headerMenu.js

```diff
@@ -84,41 +84,42 @@ const generateMenu = (menuLinks, menuName) => {
 
   return (
     <>
-      <uofg-header page-title={pageHome.title} page-url={pageHome.link.url}>{menuItems}</uofg-header>
+      {pageHome.title != null && (
```

Comment:
Should we use a strict inequality check here? `!==`

Reply:
I chose not to because I wanted the statement to check for both `null` _and_ `undefined` - if I'd used the strict inequality check, the statement would work only for `null` - at least, that's what Copilot told me :)

> In JavaScript, != and !== are two different types of comparison operators that have a subtle but important difference:
> 
> != is the non-strict equality operator. It compares the values of the two operands for equality after performing any necessary type conversions. **This means that if you compare a value with null using !=, it will also return true for undefined values because null and undefined are loosely equal.** So, pageHome.title != null will be true if pageHome.title is either null or undefined.
>
> !== is the strict non-equality operator. It compares both the value and the type of the two operands. No type conversion is done, and if the values are of different types, the values are considered unequal. So, pageHome.title !== null will only be true if pageHome.title is not exactly null. If pageHome.title is undefined, the comparison pageHome.title !== null will return false because null and undefined are of different types.

Reply:
Ahhhh, this makes sense.